### ISSUE TITLE
Mixing two operations in one statement

### ISSUE THREAD

**Issue by nkrusch** [seiller/pymwp#37]
> example3_4.c gives an error, probably because there I am mixing two operations in one statement.

[Originally posted here](https://github.com/seiller/pymwp/issues/21#issuecomment-859895794_)

The analysis makes some assumptions about left and right operand structure:

https://github.com/seiller/pymwp/blob/51bc64cfd9b3638de20a2e2671535a4975c2549f/pymwp/analysis.py#L165-L173

which prevents using the original expression  `X3 = X2 * X2 + X5;` Adding some recursive check in binary op might allow analyzing this expression in original form?

If this is not worth fixing at the moment then we can add a note in the features table that binary op is for two operands and support for multiple is in progress.

**Comment by aubertc** [comment]
Yes, both `X3 = X2 * X2 + X5;` and `X3 = X2 * (X2 + X5);` are problematic right now.
Not a big deal, but it came as a surprise.
